pkg/internal/mock/storage/mock_store.go

```diff
@@ -34,28 +36,30 @@ func (s *MockStoreProvider) Close() error {
 
 // MockStore mock store.
 type MockStore struct {
-	store map[string][]byte
-	lock  sync.RWMutex
+	Store  map[string][]byte
+	lock   sync.RWMutex
+	ErrPut error
+	ErrGet error
 }
 
 // Put stores the key and the record
 func (s *MockStore) Put(k string, v []byte) error {
 	s.lock.Lock()
-	s.store[k] = v
+	s.Store[k] = v
 	s.lock.Unlock()
 
-	return nil
+	return s.ErrPut
```

Comment:
this mockstore returns error every time?

Reply:
no